vscode-dotnet-runtime-library/src/Utils/FileUtilities.ts

```diff
@@ -12,172 +12,176 @@ import * as crypto from 'crypto';
 import { IFileUtilities } from './IFileUtilities';
 import * as lockfile from 'proper-lockfile';
 import { IEventStream } from '../EventStream/EventStream';
-import { DotnetCommandFallbackArchitectureEvent,
-   DotnetCommandFallbackOSEvent,
-   DotnetFileWriteRequestEvent,
-   DotnetLockAcquiredEvent,
-   DotnetLockAttemptingAcquireEvent,
-   DotnetLockErrorEvent,
-   DotnetLockReleasedEvent,
-   EmptyDirectoryToWipe,
-   FileToWipe,
-   SuppressedAcquisitionError
+import {
+    DotnetCommandFallbackArchitectureEvent,
+    DotnetCommandFallbackOSEvent,
+    DotnetFileWriteRequestEvent,
+    DotnetLockAcquiredEvent,
+    DotnetLockAttemptingAcquireEvent,
+    DotnetLockErrorEvent,
+    DotnetLockReleasedEvent,
+    EmptyDirectoryToWipe,
+    FileToWipe,
+    SuppressedAcquisitionError
 } from '../EventStream/EventStreamEvents';
 
 export class FileUtilities extends IFileUtilities
 {
-   public async writeFileOntoDisk(scriptContent: string, filePath: string, alreadyHoldingLock = false, eventStream? : IEventStream)
-   {
-       eventStream?.post(new DotnetFileWriteRequestEvent(`Request to write`, new Date().toISOString(), filePath));
+public async writeFileOntoDisk(scriptContent: string, filePath: string, alreadyHoldingLock = false, eventStream? : IEventStream)
```

Comment:
?? It is in the class but you un-indented it (and everything else inside the class).

Reply:
"FormatOnSave"?

Reply:
Good point, this file has always been weirdly indented. I had format on save enabled but it looks like it got disabled at some point. Thank you.